### PR TITLE
Deck Two - Extended Fore Hallway Area & Separated Stargazer Area

### PR DESCRIPTION
**Pull request by WickedCybs**
This PR does two things. First of all.

image

This entire hallway between the kitchen and bar is considered an extension of the bar, so it was linked to its power and switch and thus, started of. I replaced it with the Horizon fore hallway so the lights start on, and honestly It's probably a bit more sensible anyway.

Secondly.

image

This single APC has been powering the entire hall all along medical to operations and the entire huge hallway area by the Stargazer. It ended up being a critical fault and became a reason why atmos and power issues really screwed up this central area.

image

I added an APC here and It's considered its own area now, which will help with power management and engineering solutions.

### CODE REVIEW

**Comment by WickedCybs** [comment]
!review

File: maps/sccv_horizon/code/sccv_horizon_areas.dm [hunk 1]
```diff
@@ -357,6 +357,7 @@
 /area/horizon/hallway/deck_two/fore
 	name = "Horizon - Deck 2 - Fore Hallway"
 	icon_state = "hallF"
+
```
Comment (SleepyGemmy) [inline maps/sccv_horizon/code/sccv_horizon_areas.dm:360]: stray newline.

**Review by SleepyGemmy — CHANGES_REQUESTED** [review]
due to how jukeboxes play music (i.e. it plays to the entire area it is in), extending the fore hallway area down to where the bar area previously was is going to make the wall mounted audioconsole next to the cigarette machine play its music to the entire hallway area. the way it was previously mapped was intentional to avoid this.
removing the audioconsole is probably the easiest solution to avoid this.

File: maps/sccv_horizon/code/sccv_horizon_areas.dm (hunk 1 above)
Comment (WickedCybs) [reply maps/sccv_horizon/code/sccv_horizon_areas.dm]: Resolved.

**Comment by WickedCybs** [comment]
Removed the audio console in that hall.

**Review by SleepyGemmy — APPROVED** [review]
suggested changes addressed. mapping looks fine.

**Review by NonQueueingMatt — APPROVED** [review]

**Pull request merged**